tests/xammac_tests/xammac_tests.csproj

```diff
@@ -692,9 +692,6 @@
     <Compile Include="..\monotouch-test\ObjCRuntime\RegistrarTest.cs">
       <Link>ObjCRuntime\RegistrarTest.cs</Link>
     </Compile>
-    <Compile Include="..\api-shared\ObjCRuntime\Registrar.cs">
```

Comment:
This doesn't look quite right...

Reply:
When I built the csproj (to run the tests) in XS I got an error about duplicate files. Let me revert this here and PR it separate if I see it again.

Reply:
I just reverted this and I get this locally:
```  /Library/Frameworks/Mono.framework/Versions/5.4.1/lib/mono/msbuild/15.0/bin/Roslyn/Microsoft.CSharp.Core.targets(84,5): error MSB3105: The item "../api-shared/ObjCRuntime/Registrar.cs" was specified more than once in the "Sources" parameter.  Duplicate items are not supported by the "Sources" parameter. [/Users/donblas/Programming/macios/xcode9/xamarin-macios/tests/xammac_tests/xammac_tests.csproj]
```
doing more digging.

Reply:
```    <Compile Include="..\api-shared\ObjCRuntime\Registrar.cs">
      <Link>Registrar.cs</Link>
    </Compile>
    <Compile Include="..\monotouch-test\ObjCRuntime\RegistrarTest.cs">
      <Link>ObjCRuntime\RegistrarTest.cs</Link>
    </Compile>
    <Compile Include="..\api-shared\ObjCRuntime\Registrar.cs">
      <Link>api-shared\ObjCRuntime\Registrar.cs</Link>
    </Compile>
```

It _is_ duplicated... Why did this not fail before?

Reply:
@rolfbjarne This "works" because we use xbuild to build the tests on the bots. I used msbuild to compile time check and it fails there.

Another xbuild/msbuild ism...

Reply:
It's not duplicated here: https://github.com/chamons/xamarin-macios/blob/b520a737f3c33fc6baf836ef4921fb5e81eb1683/tests/xammac_tests/xammac_tests.csproj#L689-L697

Local changes?

Reply:
Actually nvm, of course it won't show duplicated items when I'm looking at the commit where you fixed it.

In the xcode9.2 branch it is duplicated: https://github.com/xamarin/xamarin-macios/blob/2bfbc8a834d587ebf45b4555883aaa28e57fbe5a/tests/xammac_tests/xammac_tests.csproj#L689-L697